### PR TITLE
orchestrator: a new option for enabling/disabling orchestration initi…

### PR DESCRIPTION
**Pull request by mitake**
…ally

### CODE REVIEW

File: nmz/util/config/config.go [hunk 1]
```diff
@@ -94,6 +94,10 @@ func New() Config {
 		"ethernetNFQNumber":       42,
 		"procWatchInterval":       time.Second,
 	})
+
+	// if initEnable is false, orchestrator is disabled at its initialization time
+	cfg.SetDefault("initEnable", true)
```
Comment (AkihiroSuda) [inline nmz/util/config/config.go]: nit: other options are enableFooBar rather than fooBarEnable

**Comment by mitake** [comment]
@AkihiroSuda updated the name to `skipInitOrchestrator`, PTAL

**Comment by AkihiroSuda** [comment]
lgtm
please rebase and merge?

**Comment by mitake** [comment]
@AkihiroSuda thanks, merged

**Pull request merged**